ui/app/pages/home/home.container.js

```diff
@@ -32,11 +40,29 @@ const mapStateToProps = state => {
     showPrivacyModeNotification: migratedPrivacyMode,
     shouldShowSeedPhraseReminder: !seedPhraseBackedUp && (parseInt(accountBalance, 16) > 0 || tokens.length > 0),
     isPopup,
+    threeBoxSynced,
+    restoredFromThreeBox,
+    selectedAddress,
+    threeBoxLastUpdated,
   }
 }
 
 const mapDispatchToProps = (dispatch) => ({
   unsetMigratedPrivacyMode: () => dispatch(unsetMigratedPrivacyMode()),
+  turnThreeBoxSyncingOn: () => dispatch(turnThreeBoxSyncingOn()),
+  setupThreeBox: () => {
+    dispatch(getThreeBoxLastUpdated())
```

Comment:
It is a bit awkward that we're storing the lastUpdated time of 3Box in two places - the app state and the metamask state. Moreover, the app state last updated time is only updated once per session, then never kept up-to-date after that.

Granted, it does still serve its purpose perfectly well, but it does seem confusing.

Could we just use `metamask.lastUpdated` directly, get rid of the getter method, and get rid of the app state?